src/App.js

```diff
@@ -1,27 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import Calculator from './components/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <Calculator />
+      </div>
+    );
+  }
```

Comment:
- [ ] The App.js component should be class-based. Please change this function to class-based.

Reply:
If you check properly, you will see that App.js is a class-based component

Reply:
I am sorry I mistakenly asked for this change.